mkt/webapps/tasks.py

```diff
@@ -386,33 +373,6 @@ def dump_apps(ids, **kw):
         dump_app(id)
 
 
-@task
-def zip_apps(*args, **kw):
-    today = datetime.datetime.today().strftime('%Y-%m-%d')
-    files = ['apps'] + compile_extra_files(date=today)
-    tarball = compress_export(filename=today, files=files)
-    link_latest_export(tarball)
-    return tarball
-
-
-def link_latest_export(tarball):
```

Comment:
This was called by `zip_apps()` which has not been called for a while, see comment above about `dump_apps` in `process_addons.py`.